util/rate_limiter.cc

```diff
@@ -28,15 +28,15 @@ struct GenericRateLimiter::Req {
 GenericRateLimiter::GenericRateLimiter(int64_t rate_bytes_per_sec,
                                        int64_t refill_period_us,
                                        int32_t fairness)
-    : refill_period_us_(refill_period_us),
+    : refill_period_ns_(refill_period_us * 1000),
       refill_bytes_per_period_(
           CalculateRefillBytesPerPeriod(rate_bytes_per_sec)),
       env_(Env::Default()),
       stop_(false),
       exit_cv_(&request_mutex_),
       requests_to_wait_(0),
       available_bytes_(0),
-      next_refill_us_(env_->NowMicros()),
+      next_refill_ns_(env_->NowNanos()),
```

Comment:
@yuslepukhin I assume NowNanos() on Windows should be monotonic? All we care about is time intervals here. No current wall clock is needed here.